<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>723</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,1">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,2">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label_8">
          <property name="styleSheet">
           <string notr="true">font: 24pt &quot;Adobe 黑体 Std R&quot;;</string>
          </property>
          <property name="text">
           <string>00</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,30,1">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="treeWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;Algerian&quot;;</string>
          </property>
          <column>
           <property name="text">
            <string>计算程序列表</string>
           </property>
          </column>
          <item>
           <property name="text">
            <string>机构设计</string>
           </property>
           <item>
            <property name="text">
             <string>台大设计软件</string>
            </property>
           </item>
          </item>
          <item>
           <property name="text">
            <string>齿轮计算</string>
           </property>
           <item>
            <property name="text">
             <string>涡轮计算</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>齿轮几何计算</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>齿轮计算iso</string>
            </property>
           </item>
          </item>
          <item>
           <property name="text">
            <string>轴承设计</string>
           </property>
           <item>
            <property name="text">
             <string>轴承设计</string>
            </property>
           </item>
          </item>
          <item>
           <property name="text">
            <string>连接计算</string>
           </property>
           <item>
            <property name="text">
             <string>键连接</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>螺栓连接</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>销计算</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>花键连接</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>过盈计算</string>
            </property>
           </item>
          </item>
          <item>
           <property name="text">
            <string>轴计算</string>
           </property>
           <item>
            <property name="text">
             <string>梁模型</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>轴疲劳计算</string>
            </property>
           </item>
          </item>
          <item>
           <property name="text">
            <string>带和链计算</string>
           </property>
           <item>
            <property name="text">
             <string>v带计算</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>链传动</string>
            </property>
           </item>
          </item>
          <item>
           <property name="text">
            <string>弹簧设计</string>
           </property>
           <item>
            <property name="text">
             <string>压缩弹簧</string>
            </property>
           </item>
          </item>
          <item>
           <property name="text">
            <string>螺纹传动</string>
           </property>
           <item>
            <property name="text">
             <string>螺纹传动</string>
            </property>
           </item>
          </item>
          <item>
           <property name="text">
            <string>凸轮程序</string>
           </property>
           <item>
            <property name="text">
             <string>凸轮设计</string>
            </property>
           </item>
          </item>
          <item>
           <property name="text">
            <string>公差查询计算</string>
           </property>
           <item>
            <property name="text">
             <string>配合查询</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>间隙计算</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>同轴度、对称度、圆跳动和全跳动公差</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>圆度、圆柱度公差</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>平行度、垂直度、倾斜度公差</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>直线度、平面度公差</string>
            </property>
           </item>
          </item>
          <item>
           <property name="text">
            <string>notebook打开</string>
           </property>
           <item>
            <property name="text">
             <string>notebookopen</string>
            </property>
           </item>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="4,4,4,1,1">
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QLabel" name="label_6">
          <property name="styleSheet">
           <string notr="true">font: 20pt &quot;楷体&quot;;</string>
          </property>
          <property name="text">
           <string>计算软件合集</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>测试中</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButton_2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>主程序更新</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>点击更新</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>依赖包检查安装</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>点击安装</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="pushButton_3">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>程序更新安装</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>点击安装</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>制作：lizhuang v22.04.27</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>联系方式：15736965216</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>treeWidget</sender>
   <signal>itemDoubleClicked(QTreeWidgetItem*,int)</signal>
   <receiver>MainWindow</receiver>
   <slot>childShow3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>457</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>childShow()</slot>
  <slot>childShow2()</slot>
  <slot>childShow3()</slot>
  <slot>gengxin()</slot>
  <slot>yilaibao()</slot>
  <slot>pilianggengxin()</slot>
 </slots>
</ui>
